### DISCUSSION TITLE
Enabling Mia-Assistant

### DISCUSSION THREAD

**Discussion by zachhutchins** [mia-platform/community#513] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

1.102.8

### Description

Using Mia 13.x I cannot seem to find the Mia-Assistant. Documentation labels it as being in the bottom left of the page. Feature Table in the CMS does not appear to have any mention of it as well. Where do I go to enable this feature?

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @zachhutchins, at the moment Mia-Assistant is only available in our SaaS Console.

We are working on making it also available for self hosted and it will be released in one of the next few releases.
